examples/framework.py

```diff
@@ -116,7 +125,10 @@ def configure(self):
         if self.args.ask_password:
             password = getpass.getpass()
         else:
-            password = self.config.get("global", "password")
+            if self.config.has_section(self.g_jid):
+                password = self.config.get(self.g_jid, 'password')
```

Comment:
nice idea! take care though that g_jid is a JID instance, does that work properly with ConfigParser?